jobs/cloud_controller_ng/spec

```diff
@@ -467,6 +469,10 @@ properties:
   cc.default_stack:
     default: "cflinuxfs4"
     description: "The default stack to use if no custom stack is specified for an app."
+  cc.deprecated_stacks:
+    description: |
+      List of deprecated/unsupported stack names. If a stack name is in this list, Cloud Controller bootstrap VM
+      will fail to start up if the stack is found in the database but is not explicitly listed in the stacks property.
```

Comment:
This comment is a little hard to understand. Can we rephrase it?

Reply:
Thanks @joaopapereira, updated